build.gradle

```diff
@@ -239,6 +239,7 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
     compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-openfeign'
     compile group: 'org.springframework.retry', name: 'spring-retry', version: '1.2.4.RELEASE'
+    compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.4'
```

Comment:
is this the recommended library to use for this work? based on github it does not seem a very popular choice.

Reply:
Its open source apache licence 2.0 api, most popular https://connect2id.com/products/nimbus-jose-jwt
Also its mentioned on confluence page -> https://tools.hmcts.net/confluence/display/SISM/OIDC+Integration+Guide#OIDCIntegrationGuide-ValidatingIDAMtokens